modules/cnf-installing-sriov-fec-operator.adoc

```diff
@@ -50,7 +50,7 @@ metadata:
 $ oc create -f sriov-namespace.yaml
 ----
 
-. Install the SR-IOV FEC Operator in the namespace you created in the previous step by creating the following objects:
+. Install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators in the namespace you created in the previous step by creating the following objects:
```

Comment:
Same issue with the step numbering.

Reply:
@bmcelvee I cannot see the issue here it seems to build and looks fine in the output. Can you clarify?